src/main.rs

```diff
@@ -498,6 +507,63 @@ fn run_run(cx: &Context) -> Result<()> {
     Ok(())
 }
 
+fn run_afl(cx: &Context) -> Result<()> {
+    let mut cargo = cx.cargo();
+
+    set_env(cx, &mut cargo, IsNextest(false))?;
+
+    let input_dir = "inputs/";
+    let output_dir = "outputs/";
+
+    fs::create_dir_all(input_dir)?;
+    fs::create_dir_all(output_dir)?;
+
+    if !cx.args.ignore_run_fail {
+        {
+            let mut cargo = cargo.clone();
+
+            cargo.arg("afl").arg("build");
+            cargo.arg("--target-dir").arg(cx.ws.target_dir.as_str());
```

Comment:
This needs to propagate flags passed by the user, such as releases. We probably need to create a new function like test_or_run_args in cargo.rs.